lib/layout/dummyLayoutV1.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * dummyLayoutV1.js
+ *
+ * this dumps a fake graph
+ *
+ * @author John O'Brien. 
+ */
+
+// Create a graph instance.
+
+
+function buildGraph() {
+  var lib = './../';
+  var Graph = require(lib + 'model/graph');
+  var Node = require(lib + 'model/node');
+  var Port = require(lib + 'model/port');
+  var CardinalPortPosition = require(lib + 'model/cardinalPortPosition');
+  var nodes = [];
+  //loop for building the nodes
+  for(var i =  0; i < 4; i++){
+    //creates the ports
+    var portIn = new Port( ( i * 100 ) + 01, 'input');
+    if( i === 3 ){
```

Comment:
What does 3 mean in this context? Without reading ahead a few lines it can be kind of hard to discern what's going which hurts readability a bit. I think we have an enum class in the project now, so it might be a good idea to take advantage of that.